package.json

```diff
@@ -1032,6 +1032,11 @@
                         "scope": "resource",
                         "type": "string",
                         "description": "%azureFunctions.funcCliPath%"
+                    },
+                    "azureFunctions.endOfLifeWarning": {
+                        "type": "boolean",
+                        "description": "%azureFunctions.endOfLifeWarning%",
```

Comment:
This string needs to be added to the package.nls.json file

![image](https://user-images.githubusercontent.com/12476526/199818407-eb3982b8-0c66-4258-9192-dd81634e2e6e.png)